docs/evm_compatibility/overview.md

```diff
@@ -1,37 +1,37 @@
 ---
 title: EVM Compatibility Overview
 proofedDate: na
-iterationBy: na
+iterationBy: HB
 includedInSite: true
 approvedBy: na
-comment: 404 being dealt with on link-fixer branch??
+comments: #1 we need a page on the not supported OpCodes Anton will send by slack #2 Requires supported precompiles Anton can provide by slack + explanation of WHY they are not supported #3 Requires a list of native precomiled contracts Anton can provide by slack [actually, the yellow paper https://ethereum.github.io/yellowpaper/paper.pdf is not a great source for the opcodes -- using docs instead]
```

Comment:
@afalaleev 1. is dealt with above. 2. We were going to include something on the WHY not supported 3. We have precompiles to list still